lib/hotel_manager.rb

```diff
@@ -0,0 +1,69 @@
+require 'date'
+require 'pry'
+
+module Booking
+  class Hotel_Manager
+    attr_accessor :all_reservations, :all_rooms, :available_rooms, :all_blocks
+
+    def initialize(max_rooms)
+      if max_rooms > 20
+        raise ArgumentError, "Hotel can have 20 rooms max"
```

Comment:
Hm... is there a requirement that this hotel can have less than 20 minutes? Pretty sure the requirements say that the hotel will have exactly 20 rooms :)